continuous_integration/azure/runtests.bat

```diff
@@ -0,0 +1,2 @@
+:: Run codecov if things were successful
+codecov
```

Comment:
Please rename this script to `runcodecov.bat`, but leaving a single commit in this PR.